hw/dv/tools/dvsim/sim.mk

```diff
@@ -121,7 +121,8 @@ ifneq (${sw_images},)
 				$${bazel_cquery} \
 				--ui_event_filters=-info \
 				--noshow_progress \
-				--output=starlark); do \
+				--output=starlark \
+				--starlark:expr='str(target.label)[1:] if str(target.label).startswith("@//") else target.label'); do \
```

Comment:
It would be nice if we could leave a comment nearby explaining this expression.

Unfortunately, this whole block is one giant line continuation.

Reply:
I  just pushed a new version with a comment. I use a fun/terible bash hack to put comments inside a command by using backtrick + comments:
```
echo Hello `# comment` world
```